storage/src/main/java/io/delta/storage/internal/DeltaErrors.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package io.delta.storage.internal;
+
+import scala.util.control.ControlThrowable;
+
+import java.io.IOException;
+
+public class DeltaErrors {
```

Comment:
should this be called Delta errors at all?

Reply:
just copying the existing structure that `HDFSLogStore.scala` uses (calling `DeltaErrors.scala`).

other name ideas: `ErrorHelper`, `ErrorUtil`, etc.

Reply:
@tdas what do you think?

Reply:
lets call it LogStoreErrors to mimic the same structure